.openshift/templates/ninja-board-back-end.yml

```diff
@@ -21,6 +21,14 @@ objects:
         kind: DockerImage
         name: ${DOCKER_IMAGE_REPOSITORY}
       name: latest
+- apiVersion: v1
+  kind: Secret
+  metadata:
+    labels:
+      app: ${APPLICATION_NAME}
+    name: ${SECRET_REF}
+  data:
+    WebHookSecretKey: bmluamEtYmFjaw==
```

Comment:
Should this be parameterized? Also, you can use the `stringData` field of a secret to have OpenShift/Kubernetes automatically performing a base64 encode

Reply:
Yeah, I'll make the suggested changes

Reply:
@sabre1041 I just updated the template, let me know if there are more changes to perform. Thanks